docs/source/getting_started/installation.rst

```diff
@@ -49,24 +49,17 @@ For a developer install, clone directly:
 Docker
 ------
 
-To simplify the environment setup for Composer, we provide a set of convenient Docker Images:
-
-
-============ =============== ============ ============== ===================================================================
-Linux Distro PyTorch Version CUDA Version Python Version Docker Tag
-============ =============== ============ ============== ===================================================================
-ubuntu:20.04 1.10.0          11.3.1       3.9            ``latest``, ``mosaicml/pytorch:1.10.0_cu113-python3.9-ubuntu20.04``
-ubuntu:20.04 1.10.0          cpu          3.9            ``mosaicml/pytorch:1.10.0_cpu-python3.9-ubuntu20.04``
-ubuntu:18.04 1.9.1           11.1.1       3.8            ``mosaicml/pytorch:1.9.1_cu111-python3.8-ubuntu18.04``
-ubuntu:18.04 1.9.1           cpu          3.8            ``mosaicml/pytorch:1.9.1_cpu-python3.8-ubuntu18.04``
-ubuntu:20.04 1.9.1           11.1.1       3.8            ``mosaicml/pytorch:1.9.1_cu111-python3.8-ubuntu20.04``
-ubuntu:20.04 1.9.1           cpu          3.8            ``mosaicml/pytorch:1.9.1_cpu-python3.8-ubuntu20.04``
-ubuntu:20.04 1.9.1           11.1.1       3.7            ``mosaicml/pytorch:1.9.1_cu111-python3.7-ubuntu20.04``
-ubuntu:20.04 1.9.1           cpu          3.7            ``mosaicml/pytorch:1.9.1_cpu-python3.7-ubuntu20.04``
-============ =============== ============ ============== ===================================================================
-
-Our ``latest`` image has Ubuntu 20.04, Python 3.9, PyTorch 1.10, and CUDA 11.3 and has been tested to work with
-GPU-based instances on AWS, GCP, and Azure. ``Pillow-SIMD`` is installed by default in all images.
+To simplify the environment setup for Composer, we provide a set of convenient Docker Images.
+The ``Base`` flavor of Docker Images contains PyTorch pre-installed; the ``Vision`` flavor also includes OpenCV,
+MM Segmentation, and FFCV dependencies.
+
+
+.. include:: ../../../docker/README.md
+   :parser: myst_parser.sphinx_
```

Comment:
I don't recall the details but there was a dependency (maybe for webdatasets) that was preventing us from using the version of MyST that supported this.  Just want to call it out, I'm assuming if the build passes that hopefully means this is no longer an issue.

Reply:
Yup! That was resolved -- the issue there was webdatasets used `awscli` (which had its on docutils dependency) instead of `boto3` (which doesn't have a docutils dependency).